src/adaptations/device-layer/nRF5/BLEManagerImpl.cpp

```diff
@@ -375,7 +375,11 @@ void BLEManagerImpl::DriveBLEState(void)
     WEAVE_ERROR err = WEAVE_NO_ERROR;
 
     // If the application has enabled WoBLE and BLE advertising...
```

Comment:
Please update the comment to describe the new behavior.

Reply:
Ok, added to the comment ...
Wasn't sure of the best way to interleave conditional code and extending the comment.
Let me know if you have a better suggestion.